crates/transaction-pool/src/pool/best.rs

```diff
@@ -134,6 +138,21 @@ impl<T: TransactionOrdering> crate::traits::BestTransactions for BestTransaction
     fn no_updates(&mut self) {
         self.new_transaction_receiver.take();
     }
+
+    fn skip_blob_transactions(&mut self) {
+        while let Some(pending_tx) = self.try_recv() {
+            let tx = pending_tx.transaction.clone();
+            if tx.transaction.is_eip4844() {
+                self.mark_invalid(&tx)
+            } else {
+                let tx_id = *tx.id();
+                if self.ancestor(&tx_id).is_none() {
+                    self.independent.insert(pending_tx.clone());
+                }
+                self.all.insert(tx_id, pending_tx);
+            }
+        }
```

Comment:
I think all of this should remain lazy.

instead I think this should be a simple `skip_blobs: bool` which is then used in `next()` like

```rust
fn next() {
  loop {
     let next = self.next_tx()?;
     if self.skip_blos && tx.is_eip4844() {
       // ...
     } else {
       return Some(tx)
     }
   }
}

```

Reply:
@mattsse I've just modified the implementation, tell me if this sounds good to you